From 341ad55f2ef50b279d99a8ac7b2ded2d08104d39 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Tue, 16 Jul 2019 17:29:59 +0000
Subject: [PATCH] joyent/bugview#35 JSON endpoint broken by joyent/bugview#33
 Reviewed by: Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Cody
 Peter Mello <cody.mello@joyent.com> Approved by: Cody Peter Mello
 <cody.mello@joyent.com>

---
 jirapub.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jirapub.js b/jirapub.js
index 8ac0f3a..740e206 100644
--- a/jirapub.js
+++ b/jirapub.js
@@ -408,7 +408,7 @@ handle_issue_json(req, res, next)
 			return;
 		}
 
-		mod_assert.arrayOfStrings(issue.fields.labels, 'labels');
+		mod_assert.arrayOfString(issue.fields.labels, 'labels');
 
 		if (!UNRESTRICTED &&
 		    issue.fields.labels.indexOf(CONFIG.label) === -1) {
-- 
2.21.0

